examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    with strategy.scope():
</pre><p> </p><p>Nit: Sometimes, you use 4 spaces and sometimes 2. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>